app-admin/pwman3/metadata.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+<maintainer type="person">
+	<name>Oz Tiram</name>
+	<email>oz.tiram@gmail.com</email>
+</maintainer>
+<use>
+	<flag name="mongodb">Add support for MongoDB</flag>
+	<flag name="mysql">Add support for MySQL</flag>
```

Comment:
This doesn't add anything compared to the global description.

Reply:
OK. I removed those completely from the metadata.